pkg/universe.dagger.io/x/wangwei27494731@gmail.com/kustomize/image.cue

```diff
@@ -0,0 +1,40 @@
+package kustomize
+
+import (
+	"universe.dagger.io/docker"
+	"universe.dagger.io/alpine"
+	"universe.dagger.io/bash"
+)
+
+#Image: {
+	// Kustomize binary version
+	version: *"3.8.7" | string
+
+	// Kubectl version
+	kubectlVersion: *"v1.23.5" | string
+
+	_build: docker.#Build & {
```

Comment:
Nit, you could directory embed `docker.#Build` instead of putting it in a sub key

Reply:
thanks, can you just give me an example? I don't get it.

Reply:
No problem!

See the change : 

```cue
#Image: {
	// Kustomize binary version
	version: *"3.8.7" | string

	docker.#Build & {
		steps: [
			alpine.#Build & {
				packages: {
					bash: {}
					curl: {}
				}
			},
			bash.#Run & {
				env: VERSION: version
				script: contents: #"""
					# download Kustomize binary
					curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh" | bash -s $VERSION && mv kustomize /usr/local/bin
					"""#
			},
		]
	}
}
```